stackslib/src/config/mod.rs

```diff
@@ -1656,52 +1656,231 @@ impl BurnchainConfigFile {
+    /// generated mock Stacks blocks will be saved. (pre-Nakamoto)
+    /// The path is canonicalized on load.
+    ///
+    /// Default: `None`
+    /// Deprecated: This setting was only used in the neon node and is ignored in Epoch 3.0+.
     pub mock_mining_output_dir: Option<PathBuf>,
+    /// Enable microblock mining.
+    ///
+    /// Default: `true`
+    /// Deprecated: Microblocks were removed in the Nakamoto upgrade. This setting is ignored in Epoch 3.0+.
     pub mine_microblocks: bool,
+    /// How often to attempt producing microblocks, in milliseconds (pre-Nakamoto).
+    /// Only applies when `mine_microblocks` is true and before Epoch 3.0.
+    ///
+    /// Default: `30_000`
+    /// Deprecated: Microblocks were removed in the Nakamoto upgrade. This setting is ignored in Epoch 3.0+.
     pub microblock_frequency: u64,
+    /// The maximum number of microblocks allowed per Stacks block (pre-Nakamoto).
+    ///
+    /// Default: `65535` (u16::MAX)
+    /// Deprecated: Microblocks were removed in the Nakamoto upgrade. This setting is ignored in Epoch 3.0+.
     pub max_microblocks: u64,
+    /// Cooldown period after a microblock is produced, in milliseconds (pre-Nakamoto).
+    /// Only applies when `mine_microblocks` is true and before Epoch 3.0.
+    ///
+    /// Default: `30_000`
+    /// Deprecated: Microblocks were removed in the Nakamoto upgrade. This setting is ignored in Epoch 3.0+.
     pub wait_time_for_microblocks: u64,
+    /// When operating as a miner, this specifies the maximum time (in milliseconds)
+    /// the node waits after detecting a new burnchain block to synchronize corresponding
+    /// Stacks block data from the network before resuming mining attempts.
+    /// If synchronization doesn't complete within this duration, mining resumes anyway
+    /// to prevent stalling. This setting is loaded by all nodes but primarily affects
+    /// miner behavior within the relayer thread.
+    ///
+    /// Default: `30_000`
     pub wait_time_for_blocks: u64,
-    /// Controls how frequently, in milliseconds, the nakamoto miner's relay thread acts on its own initiative
-    /// (as opposed to responding to an event from the networking thread, etc.). This is roughly
-    /// how frequently the miner checks if a new burnchain block has been processed.
+    /// Controls how frequently, in milliseconds, the Nakamoto miner's relay thread polls for work
+    /// or takes periodic actions when idle (e.g., checking for new burnchain blocks).
+    /// Default value of 10 seconds is reasonable in mainnet (where bitcoin blocks are ~10 minutes)
+    /// A lower value might be useful in other environments with faster burn blocks.
     ///
-    /// Default value of 10 seconds is reasonable in mainnet (where bitcoin blocks are ~10 minutes),
-    /// but environments where burn blocks are more frequent may want to decrease this value.
+    /// Default: `10_000` (10 seconds)
```

Comment:
Let's make this consistent -- either include the `(X seconds)` on all of the ms parameters, or on none of them.